File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {

        for (int i = 0; i < nums.size() + 1; i++)
        {
            if (find(nums.begin(), nums.end(), i) == nums.end())
            {
                return i;
            }
        }

        return 0;
    }
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        hashMap = {}
        res = []

        for eachNum in nums:
            hashMap[eachNum] = eachNum

        
        for ind in range(len(nums) + 1):
            if ind not in hashMap:
                return ind